libs/blocks/article-feed/article-feed.css

```diff
@@ -525,4 +530,4 @@ main a.article-card:any-link {
   }
 }
 
-/** END **/
+/** END **/
```

Comment:
This comment can probably be removed, but the empty line at the end of the file should be kept.